app/ModelFunctions/PhotoFunctions.php

```diff
@@ -787,4 +794,37 @@ public function getValidExtensions(): array
 	{
 		return $this->validExtensions;
 	}
+
+	/**
+	 * Central function for retrieving the metadata since this has to be called in more than one place.
+	 *
+	 * @param array  $file
+	 * @param string $path
+	 * @param string $kind
+	 * @param string $mimeType
+	 * @param string $extension
+	 *
+	 * @return void
+	 */
+	private function getFileMetadata($file, $path, $kind, $mimeType, $extension)
+	{
+		if ($kind == 'raw') {
+			$info = $this->metadataExtractor->bare();
+			$this->metadataExtractor->size($info, $path);
+			$this->metadataExtractor->validate($info);
+			$info['type'] = 'raw';
```

Comment:
#574 changes support of raw files, i.e. metadata is also extracted for raw files if possible. See https://github.com/LycheeOrg/Lychee/blob/32b460e93e56b50e0d49c25e4cf7a0e12968e282/app/ModelFunctions/PhotoFunctions.php#L286-L289

Reply:
@tmp-hallenser I adjusted my code to match the https://github.com/LycheeOrg/Lychee/pull/574